evaluators/text/BleuEvaluator/tests/test_bleuevaluator.py

```diff
@@ -25,6 +25,5 @@ def test_bleu_evaluator_caps():
 def test_bleu_evaluator_fail():
     desired = 'All cats are so super beautiful'
     actual = 'Some dogs are also cute memes'
-    with pytest.raises(Exception):
-        assert BleuEvaluator().evaluate(actual, desired) == 1.0
+    assert BleuEvaluator().evaluate(actual, desired) == 0.7351508268457205
```

Comment:
use some `almost equal` or some approximation, this test is very likely to fail in some platforms. we do not need this much precision for a test.

Reply:
check `pytest.approx`

Reply:
Got it